A42-xds-ring-hash-lb-policy.md

```diff
@@ -92,12 +92,15 @@ Here is how gRPC will handle each type of hash policy:
   the way that Envoy does.  However, we will support one special filter state
   [`key`](https://github.com/envoyproxy/envoy/blob/2443032526cf6e50d63d35770df9473dd0460fc0/api/envoy/config/route/v3/route_components.proto#L703)
   called `io.grpc.channel_id`, which will hash to the same value for all
-  requests on a given gRPC channel.  This can be used in similar situations
-  to where Envoy uses `connection_properties` to hash on the source IP address.
-  (Note that we do not recommend that applications create multiple gRPC
-  channels to the same virtual host, but if you do that, then the
-  behavior here will not be exactly the same as using `connection_properties`,
-  because each channel may use a different endpoint.)
+  requests on a given gRPC channel. In order to facilitate an even selection
+  of backends across different channels (which may or may not be in the same
+  process or machine), the value of `io.grpc.channel_id` should be initialized
+  with a uniform random number. This can be used in similar situations to where
```

Comment:
Perhaps this instead?

```suggestion
  with a random number from a uniform distribution. This can be used in similar situations to where
```

"uniform random number" could be misread as "generate a single random number and use it for all the channel ids", but I think you're trying to say that each client id should be different (and if you want a uniform distribution across backends, the random number needs to be from a uniform distribution so that hashing doesn't get skewed).

Reply:
I could see that. Changed to your wording.